app/open.js

```diff
@@ -1,4 +1,4 @@
-define(['app/database', 'jquery', 'bootstrap', 'xlsx', 'parsley', 'bootstrap-select', 'app/selectLanguage', 'app/selectSubject', 'app/forms'], function (db, $, bootstrap, XLSX, parsley, select, selectLanguage, selectSubject, forms) {
+define(['app/database', 'jquery', 'bootstrap', 'xlsx', 'parsley', 'app/select', 'app/forms'], function (db, $, bootstrap, XLSX, parsley, select, forms) {
 	var X = XLSX;
```

Comment:
You are requiring the library `xlsx`, naming that requirement `XLSX` and then storing it in the variable `X`? Why not use `X` in the define call?